src/results/outages.jl

```diff
@@ -67,13 +74,24 @@ function add_outage_results(m, p, d::Dict)
 	r["unserved_load_per_outage_kwh"] = round.(unserved_load_per_outage, digits=2)
 	r["storage_microgrid_upgrade_cost"] = value(m[:dvMGStorageUpgradeCost])
 	r["microgrid_upgrade_capital_cost"] = r["storage_microgrid_upgrade_cost"]
+    _n = ""
 	if !isempty(p.s.storage.types.elec) && Bool(round(value(m[:binMGStorageUsed]), digits=0))
 		r["storage_discharge_series_kw"] = value.(m[:dvMGDischargeFromStorage]).data
+        electric_storage_energy_capacity_kwh = round(value(m[Symbol("dvStorageEnergy"*_n)]["ElectricStorage"]), digits=2)
+        r["soc_series_fraction"] = round.(value.(m[:dvMGStoredEnergy][:,:,1:end]).data ./ electric_storage_energy_capacity_kwh, digits=3)
 	else
-		r["storage_discharge_series_kw"] = []
+		r["storage_discharge_series_kw"] = value.(m[:dvMGDischargeFromStorage]).data
+        r["soc_series_fraction"] = round.(value.(m[:dvMGStoredEnergy][:,:,1:end]).data, digits=3)  # Will be zeros
 	end
+    
+    r["critical_loads_per_outage_series_kw"] = Matrix{Array{Float64}}(undef, S, T)
```

Comment:
I'd suggest doing Array{Float64,3} instead. This would match other Outages outputs like unserved_load_series_kw and the tech dispatch series that are 3D arrays. If you want Matrix{Array} so that unused elements in the 3rd dim aren't included, then the other outputs should be changed to match this one.

Reply:
Ok, addressed in abdf8b68ea76a975440460a6a7b142d275b436f9.